.github/workflows/release_sign/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: "Cosign sign"
+description: "Signs the image using digest, and pushes the metadata to the registry"
+inputs:
+  image:
+    description: "The OCI image name. This must not include a tag or digest."
+    required: true
+    type: string
+  digest:
+    description: "The OCI image digest. The image digest of the form '<algorithm>:<digest>' (e.g. 'sha256:abcdef...')"
+    required: true
+    type: string
+  identity:
+    description: "Full oauth identity for signature verification"
+    required: true
+    type: string
+  oidc-provider:
+    description: "Specify the provider to get the OIDC token from (Optional). If unset, github-actions will be used. Options include: [spiffe, google, github-actions, filesystem, buildkite-agent]"
+    type: string
+    default: "github-actions"
+  oids-issuer:
+    description: "Full OIDS issuer URL for signature verification"
+    required: true
+    type: string
+  registry:
+    description: "The registry to login"
+    required: true
+    type: string
+  username:
+    description: "The username to registry"
+    required: true
+    type: string
+  password:
+    required: true
+    description: "The password key to fetch from secret store for registry login"
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: Docker login to registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ inputs.registry }}
+        username: ${{ inputs.username }}
+        password: ${{ inputs.password }}
+    - uses: sigstore/cosign-installer@v3.1.2
```

Comment:
There appears to be an issue with the version its trying to install from looking at the actions:

```
INFO: Downloading bootstrap version 'v2.0.0' of cosign to verify version to be installed...
      https://storage.googleapis.com/cosign-releases/v2.0.0/cosign-linux-amd64
ERROR: Unable to validate cosign version: 'v2.0.0'
Error: Process completed with exit code 1.
```

Shall we force it to use a specific version in our workflow? So something like:

```
uses: sigstore/cosign-installer@v3.1.2
with:
  cosign-release: 'v2.2.0'
```

Reply:
This is unrelated to the PR: https://github.com/rancher-sandbox/rancher-turtles/pull/247

Reply:
Ah yes.